types/hellosign-sdk/index.d.ts

```diff
@@ -1,65 +1,31 @@
-// Type definitions for hellosign-sdk 1.6
+// Type definitions for hellosign-sdk 2.1
 // Project: https://github.com/HelloFax/hellosign-nodejs-sdk
-// Definitions by: DefinitelyTyped <https://github.com/DefinitelyTyped>
+// Definitions by: David <https://github.com/dvprrsh>
+//                 Andrew <https://github.com/andrewtremblay>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 3.8
+// TypeScript Version: 4.9
```

Comment:
4.9 on this line means that the types will only work with Typescript 4.9 and above. Is that accurate? Types tend to work with any version of Typescript that's tested -- 4.1 and above currently.